src/main/java/com/javapractice/interactors/deleteTodoInteractor.java

```diff
@@ -1,24 +1,59 @@
 package com.javapractice.interactors;
 
+import java.util.ArrayList;
+
 import org.springframework.stereotype.Service;
 
 import com.javapractice.domain.Request.DeleteTodoRequest;
 import com.javapractice.domain.Response.DeleteTodoResponse;
+import com.javapractice.gateways.ILogger;
+import com.javapractice.gateways.ILoki;
 import com.javapractice.gateways.ITodoRepository;
 import com.javapractice.interactors.interfaces.IdeleteTodoInteractor;
 
 @Service
 public class deleteTodoInteractor implements IdeleteTodoInteractor {
 
     private final ITodoRepository todoRepository;
+    private final ILoki loki;
 
-    public deleteTodoInteractor(ITodoRepository todoRepository) {
+    public deleteTodoInteractor(ITodoRepository todoRepository, ILoki loki) {
         this.todoRepository = todoRepository;
+        this.loki = loki;
     }
 
+    //As a user
+    // When I delete an todo
+    // Then I want the item deleted and a log persisted to long term storage
     @Override
```

Comment:
Whenever you open a clean code architecture project, you should automatically be looking for the interactors. As a matter of convention, the constructor should have a use case in "As", "When", "Then" format.